spring-batch-geode/pom.xml

```diff
@@ -13,17 +13,25 @@
   ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
-  --><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.7.18</version>
```

Comment:
As I understand from https://github.com/spring-projects/spring-data-geode, the project is in maintenance mode, so Spring Boot 2 is probably the highest version we can reach.

Reply:
Sure!

> As I understand from https://github.com/spring-projects/spring-data-geode, the project is in maintenance mode, so Spring Boot 2 is probably the highest version we can reach.

In fact, 2.7.18 seems to be the latest OSS version available on Maven Central, so we can proceed with that.

Thank you for doing this @scordio 🙏 It's really unfortunate that we don't have a lead for this extension.. Geode is awesome!

Reply:
In hindsight, since `spring-data-geode` is [not maintained anymore](https://github.com/spring-projects/spring-data-geode?tab=readme-ov-file#spring-data-for-apache-geode-is-no-longer-maintained-as-an-open-source-project-by-broadcom-inc) and that we don't have a lead for this module, I think it's wiser to sunset this extension to save everyone's time..

Reply:
Indeed, especially being now two Spring generations behind... I'll use this PR to remove the module.